src/ad-services/anyclip/index.ts

```
@@ -25,27 +45,38 @@ export class Anyclip {
 		};
 	}
 
-	loadPlayerAsset() {
+	private setupConfig() {
+		this.pubname = context.get('services.anyclip.pubname') || 'fandomcom';
+		this.widgetname = context.get('services.anyclip.widgetname') || '001w000001Y8ud2_19593';
+		this.libraryUrl =
```

Consider writing getters for `pubname`, `widgetname` and especially `libraryUrl`. They'll get the value from the context or fallback to the hardcoded defaults. Using here getters will make the three lines of code more readable:
```ts
this.pubname = this.getPubName();
this.widgetname = this.getWidgetName();
this.libraryUrl = this.getLibraryUrl();
```
instead of:
```ts
this.pubname = context.get('services.anyclip.pubname') || 'fandomcom';
this.widgetname = context.get('services.anyclip.widgetname') || '001w000001Y8ud2_19593';
this.libraryUrl = context.get('services.anyclip.libraryUrl') || '//player.anyclip.com/anyclip-widget/lre-widget/prod/v1/src/lre.js';
```

---

Good idea, but I compressed it even more :D I always forgetting that getter are in JavaScript ;_;